examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic surface plotting with plot_surf_stat_map
+==========================================================
+
+See :ref:`plotting` for more details.
+"""
+import matplotlib.pyplot as plt
+from nilearn import plotting
+from nilearn import datasets
+
+# Retrieve the data
+nki_dataset = datasets.fetch_nki_enhanced_surface(n_subjects=1)
+
+fsaverage5_pial = nki_dataset['fsaverage_pial'][0][0]
```

Comment:
I guess that the final `[0]` means `lh`. I think we should rather make two fields `fsaverage_pial_left` and `fsaverage_pial_right`.